src/components/StatsPanel/StatsPanel.types.ts

```diff
@@ -0,0 +1,16 @@
+import type { ElementType } from 'react';
+
+export interface StatsPanelProps {
+  totalTests: number;
+  passed: number;
+  failed: number;
+  waiting: number;
+}
+
+export interface StatItemData {
+  id: string;
+  label: string;
+  value: number;
+  icon: ElementType;
+  color: string;
```

Comment:
StatItemData.color is typed as string which is too permissive given the component indexes theme.palette[color]. This will allow invalid values (like arbitrary strings) and bypass type-safety.

**Sugerencia:**
```typescript
Narrow the color type to a union of allowed palette keys, for example:

export type StatColor = 'primary' | 'secondary' | 'error' | 'warning' | 'success' | 'info' | 'text';

export interface StatItemData { ... color: StatColor; }

Or import MUI's Palette keys and use keyof Palette if you prefer a broader but typed approach.
```